common/utils.go

```diff
@@ -267,3 +267,10 @@ func ConvertFCTtoPegNetAsset(network string, asset string, userFAdr string) (Peg
 	}
 	return
 }
+
+func Abs(v int) int {
```

Comment:
:smile: http://cavaliercoder.com/blog/optimized-abs-for-int64-in-go.html golang programmers...

Reply:
Wow. I need to optimize it lol